portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -138,10 +152,22 @@ const ProjectCard = ({ initialProject }) => {
             <div className='mt-2 flex flex-wrap'>
                 <div className='self-center'>
                     {project.census} out of {project.capacity} ({project.capacity - project.census} positions left)
-                    </div>
+                </div>
+                {
+                hasJoined ?
+                <div className='p-1'>
+                    <Button text='Already Joined' onClick={() => {}} type='btnDisabled' />
+                </div>
+                :
+                hasApplied ?
+                <div className='p-1'>
+                    <Button text='Already Applied' onClick={() => {}} type='btnDisabled' />
+                </div>
+                :
```

Comment:
Might be better to leave things "blank" until `useEffect()` has done it's thing and figured out what to display, but I figure this works well for now.